third_party/move/move-compiler-v2/tests/checking/typing/recursive_local.exp

```diff
@@ -3,5 +3,5 @@ Diagnostics:
 error: unable to infer type due to cyclic type constraints (try annotating the type)
   ┌─ tests/checking/typing/recursive_local.move:5:9
   │
-5 │         x = (x, 0);
+5 │         x = vector[x];
```

Comment:
Why the change in the test case instead of adding it as a new one?

Reply:
The old one doesn't work anymore since we fixed a bug with tuple assignment and `x = (x, 0)` creates a different error then the cycle error we want to see here.